### DISCUSSION TITLE
Multivariate and multiple comparisons

### DISCUSSION THREAD

**Discussion by gsudre** [cdriveraus/ctsem#39] [Q&A] [answered]
Hi @cdriveraus and ctsem community!

I'm new to the forum and I've been very impressed with everything I've read so far about ctsem. Most examples I see describe models of uni or bivariate processes, so I was wondering about the best way to extrapolate it to scenarios with more variables. For example, say I have several brain regions (e.g. 40), and I'm interested to see their relationship to some behavioral variable. My initial approach would be to run multiple bivariate models (e.g. brain region 1 and behavior, brain region 2 and behavior, etc), and report on the meaningful models say, by analyzing DRIFT and DIFF matrices. I'd likely have to convert the estimates/SDs to p-values so I can apply some sort of multiple comparison correction?

Another approach I was toying with was to assume one or two latent processes in the brain, which are manifested in the 40 regions, and run that along with the behavioral variable. The assumption there is that whatever processes go on in the brain can be "summarized" by a handful of latent processes, but it's measured in different parts of the brain. 

Finally, I could have a big (40+1 x 40+1) matrix and try to approach it that way, but I wonder if I'll start running into convergence and/or lack of data issues.

Any thoughts?

Thanks!

**Comment by cdriveraus** [comment, marked as answer]
A system of 40 processes will definitely push things computationally and interpretationally! It's probably on the outer reaches of the 'vaguely doable with a lot of effort' spectrum. For a multiple bivariate approach, you might want to correct for multiple comparisons, use a stricter p value, or just be extra cautious in interpretation -- depends on your problem, tastes, field etc. Assuming a few dynamic factors and fitting a factor model is another way to handle the problem yeah, and again it depends on what you want to get out. 

Sorry, I know I've basically just vaguely agreed that the things you propose could be sensible. If you're only interested in relations over time, then any of those will give you something to talk about. If you have specific theoretical concerns or look for causality, you will need to be much more specific / careful regarding the modelling and interpretation.
